src/olympia/editors/tests/test_review_scenarios.py

```diff
@@ -26,7 +27,7 @@ def addon_with_files(db):
     By default the add-on is public, and the files are: beta, disabled,
     unreviewed, unreviewed.
     """
-    addon = Addon.objects.create()
+    addon = Addon.objects.create(name='My Addon', slug='my-addon')
```

Comment:
No, it's related: those tests rely on this manually created add-on and then play with the review states using the helper class I modified. The add-on had no name (which should never happen in real life), and it was causing a failure